dev/breeze/src/airflow_breeze/global_constants.py

```diff
@@ -110,9 +110,9 @@
 #  the first LTS release should be in summer/fall 2024.
 # - Innovations: Shot living releases with short support cycle - only until next Innovation/LTS release.
 # See: https://dev.mysql.com/blog-archive/introducing-mysql-innovation-and-long-term-support-lts-versions/
-MYSQL_LTS_RELEASES: list[str] = []
+MYSQL_LTS_RELEASES: list[str] = ["8.4"]
 MYSQL_OLD_RELEASES = ["8.0"]
-MYSQL_INNOVATION_RELEASE = "8.3"
+MYSQL_INNOVATION_RELEASE: str | None = None
```

Comment:
Does this mean we no longer support 8.3 at all, in favor of the 8.4 LTS?

Reply:
Technically yep we do not support, at least do not test it under the `8.3`, as well as `8.2`, `8.1`. In practice it mean if Airflow will able to run under the 8.4 it should also run under 8.1-8.3 with high probability. In addition Innovation release could be upgraded up until the next LTS release.

Innovation release more handy for development purpose - detect some removed/deprecated stuff rather than Production usage, however it is GA and Production ready according to this [announcement](https://dev.mysql.com/blog-archive/introducing-mysql-innovation-and-long-term-support-lts-versions/)

Reply:
That all makes sense.  Just wanted to make sure.